src/components/ModalPayment/ModalCardDetails/ModalCardDetails.tsx

```diff
@@ -264,15 +307,42 @@ const ModalCardDetails = ({
               ))}
             </div>
             <br />
-            <Subheader>Checkout details</Subheader>
+            <Subheader>{setDetailsText(purchaseType, amount)}</Subheader>
+
             <span>
               {' '}
-              {purchaseTypePhrase(false, purchaseType)} of{' '}
+              {purchaseTypeMessage(purchaseType, amount)} of{' '}
               <b>
                 ${amount} {numberOfMealsText}
               </b>{' '}
               to {sellerName}{' '}
             </span>
+
+            {lucData.firstName !== '' && (
+              <span>
+                <br />
+                <br />
+                {t('modalPayment.modalCardDetails.message.luc_name')}
+                <b>
+                  {`${lucData.firstName} 
+                  ${
+                    lucData.middleInitial.length > 0
+                      ? lucData.middleInitial.substring(0, 1).toUpperCase() +
+                        '. '
+                      : ''
+                  } 
+                  ${lucData.lastName}`}
+                </b>
```

Comment:
Bold styling in CSS is best used in this case since it's not adding any sort of emphasis on a particular word/offering any sort of structural advantage. Here is a good summary of when to use CSS/HTML bold tags.

https://stackoverflow.com/questions/3908758/use-of-native-html-elements-vs-css

It's also to note the difference between bold and strong in HTML5: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong